package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "keywords": [],
   "author": "",
-  "license": "ISC",
+  "license": "MIT",
   "scripts": {
     "clean": "rimraf build",
```

Comment:
IMHO version shouldn't be 1.0.0 I would make it 0.1.0 we need to massage it a bit maybe (new versions will come)

Reply:
Done.